pkgs/by-name/en/encore/go.nix

```
@@ -0,0 +1,81 @@
+{ fetchFromGitHub, stdenv, lib, go, ... }:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "go-encore";
+  version = "1.22";
```

Version is missing the patch number as published on https://github.com/encoredev/go/releases/tag/encore-go1.22.0
```suggestion
  version = "1.22.0";
```